dev-vcs/tig/tig-2.5.8.recipe

```diff
@@ -63,7 +63,12 @@ BUILD()
 INSTALL()
 {
 	make install
-	make install-doc-man
+
+	# temporary solution because `make install-doc-man` fails/works randomly.
```

Comment:
Works with cmd:xmlto and cmd:asciidoc as mentioned on IRC, also mentioned it doesn't bring anything new, so let's go this way. :)